documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -1,33 +1,32 @@
 [[raspi-config]]
 == `raspi-config`
 
-TIP: Raspberry Pi Desktop users can access a graphical version of this application at **Preferences** > **Raspberry Pi Configuration**. However, some advanced configuration is only available in `raspi-config`.
+`raspi-config` helps you configure your Raspberry Pi. Changes to `raspi-config` will affect xref:config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] and various Linux configuration files.
```

Comment:
Could change "various Linux configuration files" to "various other configuration files" ? :shrug:

Reply:
Maybe even simpler will do?

```suggestion
`raspi-config` helps you configure your Raspberry Pi. Changes to `raspi-config` will affect xref:config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] and other configuration files.
```

Reply:
I guess you _could_ even change "affect" to "modify" ? :shrug: